refiner-lib/src/refiner_inner.rs

```diff
@@ -959,6 +959,17 @@ enum RefinerError {
     FailNearTx,
 }
 
+impl fmt::Debug for RefinerError {
```

Comment:
Why manually provide this implementation?

Reply:
To be honest, to just satisfy the compiler only. The compiler [doesn't like](https://github.com/aurora-is-near/borealis-engine-lib/actions/runs/9190831643/job/25275943575#step:4:1428) that we don't use values in the enum variants.